src/netmanager-app/components/NetManagerMap/page.tsx

```
@@ -0,0 +1,386 @@
+                          type: 'geojson',
+                          data: geojsonData
+                        });
+                       
+                        // Listen for when the mouse enters a feature in your 'data-layer'
+                        mapRef.current.on('mouseenter', 'data-layer', (e) => {
+                                if (mapRef.current) {
+                                    mapRef.current.getCanvas().style.cursor = 'pointer';
+                                }
+                                // Get the feature under the mouse pointer (the first feature).
+                                const feature = e.features ? e.features[0] : null;
+                                if (feature && feature.properties) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)